File: documentation-examples-scala/src/scala/dev/typr/foundationssc/docs/db2/BinaryTypes.scala
package dev.typr.foundationssc.docs.db2
import dev.typr.foundationssc.*
import dev.typr.foundationssc.data.*

@SuppressWarnings(Array("unused"))
object BinaryTypes:
  // start
  val binaryType: Db2Type[Array[Byte]] = Db2Types.binary
  val binary16: Db2Type[Array[Byte]] = Db2Types.binaryOf(16)
  val varbinaryType: Db2Type[Array[Byte]] = Db2Types.varbinary
  val blobType: Db2Type[Array[Byte]] = Db2Types.blob
  // stop

File: documentation-examples-kotlin/src/kotlin/dev/typr/foundationskt/docs/db2/BinaryTypes.kt
package dev.typr.foundationskt.docs.db2

import dev.typr.foundationskt.*
import dev.typr.foundationskt.data.*

@Suppress("unused")
class BinaryTypes {
    //start
    val binaryType: Db2Type<ByteArray> = Db2Types.binary
    val binary16: Db2Type<ByteArray> = Db2Types.binaryOf(16)
    val varbinaryType: Db2Type<ByteArray> = Db2Types.varbinary
    val blobType: Db2Type<ByteArray> = Db2Types.blob
    //stop
}
